### DISCUSSION TITLE
ERROR: Could not build wheels for cryptography which use PEP 517 and cannot be installed directly

### DISCUSSION THREAD

**Discussion by Umbys** [indomus/forum#847] [GENERALE - Discussioni generiche]
Salve,
sto installando Home Assistant su Raspberry con installato Rasbian.

Dopo l'installazione del sistema Rasbian, procedo con l'installazione di Home Assistant,
(in automatico installa la 2021.5). L'installazione va a buon fine, Home Assistant funziona ma mi indica che la versione
di Pyton (la 3.7.5 per esattezza) non è sopportata e devo upgradare alla versione 3.8.0 osuperiore.

Così seguo la guida per upgradare Pyton alla 3.8.6. Funziona ancora tutto, dopo il restart del raspberry,
posso upgradare Home Assistant alla versione 2021.10, ma quando do il comando "pip3 install homeassistant" mi fallisce con questo errore:

[Errore HA.txt](https://github.com/indomus/forum/files/7314755/Errore.HA.txt)

Dov'è che sbaglio?

Grazie infinite

**Comment by snakuzzo** [comment]
Leggi [QUI](https://indomus.it/news/hot-aggiornamento-2021-10-per-home-assistant-core/?ht-comment-id=2930948)
